docs/docs/deploying-to-digitalocean-droplet.md

```diff
@@ -0,0 +1,238 @@
+   ```
+
+3. Install npm
+
+   ```bash
+   sudo apt-get install npm
+   ```
+
+   To view the version of node and npm installed, run,
+
+   ```bash
+   node -v
+   npm -v
+   ```
+
+4. To install the latest stable release using the `n` package (Optional) (Recommended),
+
+   ```bash
+   sudo npm install -g n
+   sudo n stable
+   ```
+
+   `Note`: If you check the version now, you would see the older versions of node and npm from the cache.
+   You can either exit and restart your terminal or refresh the cache by following commands,
+
+   ```bash
+   hash node
+   hash npm
+   ```
+
+5. Install the gatsby-cli now globally. This will be useful ahead in building the Gatsby site for production.
+
+   ```bash
+   sudo npm install -g gatsby-cli
+   ```
+
+### Clone your repository to the droplet
+
+The next step is to clone the repository containing your Gatsby app (Replace `<your-github-repo-site>` with your Github repository link)
+
+```bash
+git clone <your-github-repo-site>
+```
+
+> Note: this guide will refer to the cloned directory as `<my-gatsby-app>` for simplicity.
+
+### Build your gatsby site to generate a static site for production
+
+The static files will be hosted publicly on the droplet. The `gatsby build` command provides utility to build the site and generate the static files in the `/public`.
+
+1. Install dependencies.
+
+```bash
+cd my-gatsby-app
+sudo npm install
+```
+
+2. Run build to generate static files.
+
+```bash
+sudo gatsby build
+```
+
+### Install Nginx Web Server to host the static site and open firewall to accept HTTP and HTTPS requests
+
+To host a website or static files onto a Linux-based server/VPS, a web-server like Apache or Nginx is required.
+
+Nginx is web-server. It provides the infrastructure code for handling client requests from the World Wide Web, along with features like a load balancer, mail proxy, and HTTP Cache.
+
+1. Install Nginx.
+
+   ```bash
+   sudo apt-get install nginx
+   ```
+
+2. Configure firewall settings of the droplet to listen to HTTP and HTTPS requests on port 80 and 443 respectively.
+
+   ```bash
+   sudo ufw allow 'Nginx HTTP'
+   sudo ufw allow 'Nginx HTTPS'
+   ```
+
+3. To check the access,
+
+   ```bash
+   sudo ufw app list
+   ```
+
+4. If `ufw` status is disabled/inactive, you can enable it with the following command:
+
+   ```bash
+   sudo ufw enable
+   ```
+
+   Allow the OpenSSH if not already done, to not disconnect from your droplet.
+
+   ```bash
+   sudo ufw allow 'OpenSSH'
+   ```
+
+### Configuring Nginx to point to your Gatsby site's `/public` directory and add your domain
+
+Change the root directory configuration of Nginx in the default server block file
+
+1. Go to `/etc/nginx/sites-available/`
+
+   ```bash
+   cd /etc/nginx/sites-available/
+   ```
+
+2. Open the file default
+
+   ```bash
+   sudo vim default
+   ```
+
+3. Edit the file and make the following changes for below-mentioned fields, leave the rest of the fields as is.
```

Comment:
```suggestion
3. Edit the file and make the following changes for below-mentioned fields, leave the rest of the fields as is. Your exact path may vary, but it may resemble `/home/<your non-root user>/<my-gatsby-app>/public`.
```

People might need a little help finding the path, I had to exit out of Vim to find it since I guessed wrong.